server/docs/api.md

```diff
@@ -93,3 +95,88 @@ $ curl -d '{
     }
 }
 ```
+
+### POST /api/like
+
+**Example Request**
+
+```
+$ curl -d '
+{
+    "review_id": "60eec6d2054f49d2c58d3390"
+}
+' -H 'Content-Type: application/json' -x POST https://bookworms-api.vercel.app/api/like 
+
+```
+
+**Example Response**
+
+```
+201 OK
+
+
+{
+    "success": true,
+    "data": {
+        "id": "60eec6d2054f49d2c58d3390",
+        "user_name": "test_user",
+        "book_name": "Harry Potter",
+        "text": "I'm sure everyone has their own childhood memories of reading the Harry Potter books that they'll tell their grandkids about, but I'll never forget going to see the first movie in the theaters",
+        "categories": [
+            "Animals",
+            "Plantation"
+        ],
+        "created_at": "2021-07-14T11:13:22.619Z",
+        "updated_at": "2021-07-14T11:13:22.619Z",
+        "likes": 1
+    },
+    "time_stamp": "Wed Jul 14 13:14:23 +0200 2021"
+}
+```
+
+### GET /api/filter
+
+**Example Request**
+
+```
+$ curl https://bookworms-api.vercel.app/api/filter\?category\=Plantation 
```

Comment:
Better to use `categories` instead of `category` as it is a comma separated list of categories